packages/react-toolbar/Spec.md

```diff
@@ -2,62 +2,100 @@
 
 ## Background
 
-_Description and use cases of this component_
+### Definition
+
+`Toolbar` is a surface that houses commands that operate on the content of the window, panel, or parent region it resides above. `Toolbar` is one of the most visible and recognizable way to surface commands, and can be an intuitive method for interacting with content on the page; however, if overloaded or poorly organized, they can be difficult to use and hide valuable commands from your user. `Toolbar` can also display a search box for finding content, hold simple commands as well as menus, or display the status of ongoing actions.
 
 ## Prior Art
 
-_Include background research done for this component_
+As a part of the spec definitions in Fluent UI, a research effort has been made through [Open UI](https://open-ui.org/). The current research proposal is available as an open source contribution undergoing review ([research proposal](https://github.com/openui/open-ui/pull/452))
+
+## Comparison of `@fluentui/react` and `@fluentui/react-northstar`
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- All mentions of v7 or v8 == `@fluentui/react` ([docsite](https://developer.microsoft.com/en-us/fluentui#/))
+- All mentions of v0 == `@fluentui/react-northstar` ([docsite](https://fluentsite.z22.web.core.windows.net/))
 
-## Sample Code
+The main difference between `@fluentui/react`'s `CommandBar` and `@fluentui/react-northstar`'s `Toolbar` is the right group of commands present in `CommandBar` which will not remove items on overflow.
```

Comment:
This line doesn't make sense to me. Does one of these controls remove right items when overlflowing? Is that the Toolbar?

Reply:
I think that was my confusion, the v8 `CommandBar` has right group of items but doesn't remove it on overflow. But following next spec for converged command bar that is something that will happen.

Reply:
can you please briefly describe, how we will create CommandBar in the future, probably by composing Toolbar and Overflow?